src/jdk.jpackage/share/classes/jdk/jpackage/internal/CLIHelp.java

```diff
@@ -46,16 +48,15 @@ public static void showHelp(boolean noArgs) {
         if (noArgs) {
             Log.info(I18N.getString("MSG_Help_no_args"));
         } else {
-            Platform platform = (Log.isVerbose()) ?
-                    Platform.UNKNOWN : Platform.getPlatform();
+            OperatingSystem platform = OperatingSystem.current();
```

Comment:
Should we add if (Log.isVerbose) back? Otherwise default case in switch statement never executed.

Reply:
I was puzzled by that check early on and its still a puzzle. It seems a bit of a hack overloaded on the platform dispatch.
As it turns out, the only time that `Log.isVerbose()` is true when `showHelp()` is called is when the environment variable `JPACKAGE_DEBUG` is used for debugging.
`showHelp()` is only called before argument parsing is done and it doesn't look for --verbose. See `Main.execute()` 
So checking for isVerbose has no practical function.

Reply:
Makes sense. Looks good.